src/api/get-installations.js

```diff
@@ -59,6 +64,8 @@ async function findWindowsBrowsers () {
     if (edgeAlias)
         installations['edge'] = edgeAlias;
 
+    await exec(`chcp ${origCodePage}`);
```

Comment:
Well, I see now. We have an incosistency here, I can break things by calling `detectMicrosoftEdge` directly. What we need to do here: create `setCmdCodePage()` and `restoreCmdCodePage` methods and call them from `getWindowsBrowser` to make things more  explicit.